application/frontend/controllers/UserController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace frontend\controllers;
+
+use common\models\User;
+use frontend\components\BaseRestController;
+use Yii;
+use yii\web\NotFoundHttpException;
+
+class UserController extends BaseRestController
+{
+    public function actionCreate(): User
+    {
+        $existingUser = User::findOne([
+            'employee_id' => Yii::$app->request->getBodyParam('employee_id')
+        ]);
+
+        $user = $existingUser ?? new User();
+
+        $user->attributes = Yii::$app->request->getBodyParams();
```

Comment:
As long as we have unit/api tests to ensure unsafe attributes cannot be set I'm okay with it. But we need multiple tests for each scenario to be sure it works properly.